src/tests/ftest/launch.py

```diff
@@ -1134,19 +1153,50 @@ def check_big_files(avocado_logs_dir, task, test_name, args):
             cdata.extend(["  {}".format(big_file) for big_file in big_files])
     if cdata:
         destination = os.path.join(avocado_logs_dir, "latest")
+        message = "Log size has exceed threshold for this test on: {}".format(
+            hosts)
         status = create_results_xml(
-            hosts, test_name, "\n".join(cdata), destination)
+            message, test_name, "\n".join(cdata), destination)
     else:
         print("No log files found exceeding {}".format(args.logs_threshold))
 
     return status
 
 
-def create_results_xml(hosts, testname, output, destination):
-    """Create Junit xml file.
+def report_skipped_test(test_file, avocado_logs_dir, reason):
+    """Report an error for the skipped test.
+
+    Args:
+        test_file (str): the test python file
+        avocado_logs_dir (str): avocado job-results directory
+        reason (str): test skip reason
+
+    Returns:
+        bool: status of writing to junit file
+
+    """
+    message = "The {} test was skipped due to {}".format(test_file, reason)
+    print(message)
+
+    # Generate a fake avocado results.xml file to report the skipped test.
+    # This file currently requires being placed in a job-* subdirectory.
+    test_name = get_test_category(test_file)
+    time_stamp = datetime.now().strftime("%Y-%m-%dT%H.%M")
+    destination = os.path.join(
+        avocado_logs_dir, "job-{}-da03911-{}".format(time_stamp, test_name))
+    try:
+        os.makedirs(destination)
+    except (OSError, FileExistsError) as error:
+        print("Failed to create {}: {}".format(destination, error))
+    return create_results_xml(
+        message, test_name, "See console log for more details", destination)
```

Comment:
Is `console.log` here `job.log`?  I'm concerned that most people won't know which actual log file is the _console.log_.

Reply:
No, it's the console log.  The avocado `job.log` does not contain any launch.py output.

Reply:
One thing I was debating implementing was including the actual error from the failed clush rm command, but there were a few concerns:

1. Every skipped test would be reporting the same error, which is a bit excessive and could seem to imply multiple failed communication attempts when there really was only one.
2. We could report a single error and list all the tests that were skipped with a single reference to the failure, but we would need either need to create a fake (or just use one of the failed test case's) job results directory to place the JUnit file.  This makes comparing results from different runs a bit wonky as the number of tests (as indicated by the jog-results directories and the existence of an *.xml file) would fluctuate.
3. Obtaining the command result would require a bit of a rewrite of `clean_logs` and `spawn_commands` which would affect a bunch of code, for what is I suspect an infrequent issue.

Reply:
> No, it's the console log.

Ahhh.  The:

![image](https://user-images.githubusercontent.com/559317/97896938-c3156a00-1d03-11eb-903f-82ed08bea3b0.png)

log, which is https://build.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Fdaos/detail/PR-3699/9/pipeline for the test run in this PR.  We unfortunately lack nomenclature for that log.  I tend to call it the "main stage log" but I don't know how well that would be understood either.

> We do not know which stage we are in (Functional on CentOS 7, Functional Hardware Small, etc.), so providing a little more direction is limited.

We could know, either (possibly) through setting an environment variable in `Jenkinsfile` that `launch.py` could pick up, or if an environment variable from `Jenkinsfile` is not doable, an argument to ftest.sh that sets an environment variable for `launch.py`.  This would not be the first time I have considered `ftest.sh`/`launch.py` being more useful if it knew which stage it was executing.

> Also if this is run manually the data is in the console (stdout) output.

The environment variable, or rather lack thereof could also be used to help direct a non-CI/manual running user.

I'm on the fence about whether this is [feeping creaturism](https://en.wikipedia.org/wiki/Feature_creep#Feeping_creaturism) or if it is a legitimately useful enhancement to try to give developers as much info as possible and reduce the channel questions/chatter about which log is actually meant.

> One thing I had debated implementing was including the actual error from the failed clush rm command, but there were a few concerns:

Hrm.  You have me on the fence about this.  I am always in favour of trying to avoid directing users to log files.  Particularly when the redirection could be vague like this one.  But I do agree with your concerns.  I agree with concern 2 so that is not even an option IMHO.   Concern 1 could be workable, perhaps with a bit of wording such that it's clear that a given failure is the result of a previous failure, but that doesn't resolve concern 3.

Reply:
One option would be to refer to it as the "launch.py command output" instead of the console log.  In terms of other improvements, i.e. knowing which stage is being run and providing details about the error, I think these could be addressed in a future update.  For now, knowing that we skipped tests is more important and until we're notified of such we will not have a good sense of how often this situation occurs.

Reply:
> One option would be to refer to it as the "launch.py command output" instead of the console log.

I'm not confident that everyone knows the functional testing infrastructure to know what that is.  Let's go with it, although, I'd really love to refer to it as exactly the step in the stage in the blue ocean stage view.

> In terms of other improvements, i.e. knowing which stage is being run and providing details about the error, I think these could be addressed in a future update. For now, knowing that we skipped tests is more important and until we're notified of such we will not have a good sense of how often this situation occurs.

Fair enough.